products/express.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Express
+category: framework
+tags: javascript herodevs
+iconSlug: express
+permalink: /express
+releasePolicyLink: https://expressjs.com/en/support/
+changelogTemplate: https://github.com/expressjs/express/releases/tag/__LATEST__
+releaseDateColumn: true
+eolColumn: Security Support
+eoesColumn: Commercial Support
+
+auto:
+  methods:
+    - git: git@github.com:expressjs/express.git
+      regex: ^v(?P<version>\d+\.\d+\.\d+)$
+      template: "{{version}}"
```

Comment:
```suggestion
```

[The default regex](https://github.com/endoflife-date/release-data/blob/d71df51842a77040ab11e3671dc9a231fd5aefd6/src/common/endoflife.py#L13), that supports version both with and without the v prefix, should be enough ?